sdk/src/test/java/com/mapbox/maps/StyleTest.kt

```diff
@@ -348,4 +354,10 @@ class StyleTest {
     style.setStyleTerrainProperty("id", value)
     verify { nativeMap.setStyleTerrainProperty("id", value) }
   }
+
+  @Test
+  fun removeStyleTerrain() {
+    style.removeStyleTerrain()
```

Comment:
can I as a user call it twice?
what will happen? or it's not our responsibility

Reply:
it's not our responsibility.